boards/xtensa/heltec_wireless_stick_lite_v3/doc/index.rst

```diff
@@ -0,0 +1,306 @@
+   | J3.5   | GPIO48  |                             |
+   +--------+---------+-----------------------------+
+   | J3.6   | GPIO0   | User Switch                 |
+   +--------+---------+-----------------------------+
+   | J3.7   | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.8   | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.9   | U1RXD   | UART 1                      |
+   +--------+---------+-----------------------------+
+   | J3.10  | GPIO21  |                             |
+   +--------+---------+-----------------------------+
+   | J3.11  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.12  | U1TXD   | UART 1                      |
+   +--------+---------+-----------------------------+
+   | J3.13  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.14  | NC      | Reset Switch                |
+   +--------+---------+-----------------------------+
+   | J3.15  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.16  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.17  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.18  | TBD     |                             |
+   +--------+---------+-----------------------------+
+   | J3.19  | TWAI_TX | CAN (optional)              |
+   +--------+---------+-----------------------------+
+   | J3.20  | TWAI_RX | CAN (optional)              |
+   +--------+---------+-----------------------------+
+
+
+System requirements
+*******************
+
+Prerequisites
+=============
+
+Espressif HAL requires WiFi and Bluetooth binary blobs in order work. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west blobs fetch hal_espressif
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+Programming and Debugging
+*************************
+
+ESP-IDF bootloader
+==================
+
+The board is using the ESP-IDF bootloader as the default 2nd stage bootloader.
+It is build as a subproject at each application build. No further attention
+is expected from the user.
+
+MCUboot bootloader
+==================
+
+User may choose to use MCUboot bootloader instead. In that case the bootloader
+must be build (and flash) at least once.
+
+There are two options to be used when building an application:
+
+1. Sysbuild
+2. Manual build
+
+.. note::
+
+   User can select the MCUboot bootloader by adding the following line
+   to the board default configuration file.
+   ```
+   CONFIG_BOOTLOADER_MCUBOOT=y
+   ```
+
+Sysbuild
+========
+
+The sysbuild makes possible to build and flash all necessary images needed to
+bootstrap the board with the EPS32S3 SoC.
```

Comment:
Correct spelling would be "ESP32-S3" but it looks like this generic section of the documentation is typically just using the term "ESP32" for all the other ESP32 boards.

```suggestion
bootstrap the board with the EPS32-S3 SoC.
```